k8s/controllers/workflow/activation_controller.go

```diff
@@ -54,6 +55,11 @@ func (r *ActivationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	//get Activation
 	activation := &workflowv1.Activation{}
+
+	resourceK8SId := activation.GetNamespace() + "/" + activation.GetName()
+	operationName := fmt.Sprintf("%s", constants.ActivationOperationNamePrefix)
```

Comment:
I prefer you determine operationName at the beginning otherwise it is hard to maintain.

Reply:
Sure. Moved to the beginning of the reconcile.